examples/python/generic_mle.py

```diff
@@ -103,13 +103,13 @@ def _ll_nb2(y, X, beta, alph):
 class NBin(GenericLikelihoodModel):
     def __init__(self, endog, exog, **kwds):
         super(NBin, self).__init__(endog, exog, **kwds)
-        
```

Comment:
@josef-pkt would fixing this be more palatable if also fixed in the corresponding ipynb file?